src/tests/contract_test/utils.cljs

```diff
@@ -12,3 +12,11 @@
                        :params     args
                        :on-success p-resolve
                        :on-error   p-reject}))))
+
+(defn get-main-account
```

Comment:
Btw main account is not good to use because it could actually be the 'chat' account. You should just use that method use default account to guarantee it's related to wallet